client/src/pages/Experience/Experience.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+import SignUpContainer from "../../components/SignUpContainer";
+import NewExperienceForm from "./NewExperienceForm";
+import AddExperienceButton from "./AddExperienceButton";
+
+// TO-DO: ADD FUNCTIONALITY TO FILTER LANGUAGES ALREADY SELECTED
+const Experience = props => {
+  const [experience, setExperience] = useState([{ C: 1 }]);
+
+  const languagesAvailable = [
+    "C",
+    "C++",
+    "Java",
+    "JavaScript",
+    "Python",
+    "Ruby"
```

Comment:
can we put this in a utils folder or something... imagine we will use it throughout